CHANGELOG.md

```diff
@@ -0,0 +1,330 @@
+# _fastlane_ CHANGELOG
+
+---
+
+When contributing to fastlane, don't forget to add an entry to this CHANGELOG.md file by copy/pasting and adapting this entry template:
+
+```
+* Describe your change here. End it with a full-stop and 2 spaces.  
```

Comment:
Not a native english speaker, but that's how I was told to call a punctuation period mark, `.`, used to end a sentence :wink: